src/LookupTable/hooks/useGetStructureHooks.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react'
```

Comment:
Simple hooks to fetch from endpoints instead of GraphQL query hooks.

Reply:
Would it be more direct to pass a method for refetch ? 

https://github.com/msupply-foundation/conforma-web-app/compare/275T-data-table-tweaks...275T-data-table-tweaks(Review-suggestion)

Reply:
Yeah, good call. Thanks: 424b4ee6